examples/nextjs-ssr/README.md

```diff
@@ -0,0 +1,21 @@
+# sgID Next.js (SSR) Example
+
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+Before you can run the development server, you will have to register your client on the sgID [developer portal](https://developer.id.gov.sg/).
+
+> For more information about sgID, please visit the [developer documentation](https://docs.id.gov.sg/).
+
+Copy the `.env.example` file, rename it to `.env`, and fill in your credentials obtained during registration.
```

Comment:
There is no `.env.example` file

Reply:
Related: do we want to use `dotenv`? I'm thinking it'll make things easier for them but happy to hear your thoughts

Reply:
hm what do you mean by use `dotenv`? Next.js support environment variables out of the box so I don't really understand why we would need to use `dotenv`. could you help to clarify?

Reply:
Ahh ok, no issues then, just another instance of my lack of knowledge re: Next.js showing haha

Reply:
Next.js is making people dumb because they include everything ootb haha!